app/src/integrationTests/ContribIntegrationTests.tsx

```diff
@@ -235,8 +235,13 @@ class ContributionPortalIntegrationTests extends IntegrationTestSuite {
                 expect(payload).to.eql(JSON.parse(`{
                     ":group-id": "${groupId}",
                     ":touchstone-id": "${touchstoneId}",
-                    ":scenario-id": "${scenarioId}"
+                    ":scenario-id": "${scenarioId}"                    
                 }`));
+                const query = QueryString.parse(decoded.query);
+                expect(query).to.eql(JSON.parse(`{
+                    "redirectUrl": "http://localhost:5000/redirect/back"                   
+                }`
```

Comment:
Why are you parsing this from a string, instead of just using an object literal?

Reply:
I copied and pasted from the existing test tbh, haven't checked whether its necessary